lib/LocalAnomaly.js

```diff
@@ -16,6 +16,7 @@
 "use strict";
 var NODEJS = ((typeof module !== 'undefined') && module.exports);
 var PATH = (NODEJS) ? "./" : "";
+var TEXT_TYPES = ["categorical", "text"];
```

Comment:
you could add here "datetime"... usuallly those fields are excluded automatically, but i guess a sufficiently determined user could end up adding them...